scripts/check_path_modified.sh

```diff
@@ -25,9 +25,11 @@ else
     DIFF_SOURCE="remotes/origin/master"
 fi
 
-if git diff --exit-code $DIFF_SOURCE..HEAD -- ${CHECK_PATH}; then
+if git diff --exit-code $DIFF_SOURCE..HEAD -- ${CHECK_PATH} && test -e ${CHECK_PATH}; then
```

Comment:
This is added because paths that don't exist are marked as not modified. This has implications for the ARM prairielearn PR, in which I don't want to special case the `check_path_modified` (It would currently check `images/prairielearn`)